src/libecalc/core/models/compressor/results.py

```diff
@@ -249,7 +267,21 @@ def from_result_list_to_dto(
                 for t in range(len(result_list))
                 if result_list[t].stage_results[i].outlet_actual_rate_m3_per_hour is not None
             ]
-            outlet_stream_condition[i].actual_rate_before_asv_m3_per_hr = [np.nan] * len(result_list)
+            outlet_stream_condition[i].actual_rate_before_asv_m3_per_hr = [
+                result_list[t].stage_results[i].outlet_actual_rate_m3_per_hour
+                for t in range(len(result_list))
```

Comment:
if result_list[t].stage_results[i].outlet_stream is not None
                and result_list[t].stage_results[i].standard_rate_before_asv_sm3_per_day is not None
                else np.nan
                for t in range(len(result_list))